src/BuildCheck.UnitTests/EndToEndTests.cs

```diff
@@ -31,11 +31,72 @@ public EndToEndTests(ITestOutputHelper output)
 
     public void Dispose() => _env.Dispose();
 
-    [Theory(Skip = "https://github.com/dotnet/msbuild/issues/10036")]
```

Comment:
note: bring this skip back for this test and add it for `SampleAnalyzerIntegrationTest_AnalyzeOnBinaryLogReplay`. 
For now I left them like this to demonstrate that the tests are passing